test/UnitTest/TestingControllers/InstancesControllerTests.cs

```diff
@@ -622,6 +623,88 @@ public async Task GetEmptyListOfInstances_UserRequestsInstancesNoPartyIdDefinedA
             registerService.VerifyAll();
         }
 
+        /// <summary>
+        /// Test case: Get Multiple instances with an organisation number(which for some reason ended up in bad request in the register) and without specifying instance owner partyId.
+        /// Expected: Register service throws 400 bad request.
+        /// </summary>
+        [Fact]
+        public async Task GetMany_UserRequestsInstancesNoPartyIdDefinedAndWithOrganisation_ThrowsBadRequest()
```

Comment:
Returns, not throws.
I would probably rename to GetMany_RegisterServiceThrowsBadRequestException_ReturnsBadRequest.

Reply:
As the controller is returning bad request and this is a test for controller then it should be returns.